automation/jleechanorg_pr_automation/automation_safety_manager.py

```
@@ -0,0 +1,821 @@
+            email_to = os.environ.get("EMAIL_TO")
+            username, password = self._get_smtp_credentials()
+            return bool(smtp_server and smtp_port and email_to and username and password)
+        except Exception:
+            return False
+
+
+def main():
+    """CLI interface for safety manager"""
+
+    parser = argparse.ArgumentParser(description="Automation Safety Manager")
+    parser.add_argument("--data-dir", default="/tmp/automation_safety",
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp/automation_safety"

(S108)